CHANGELOG.md

```diff
@@ -16,6 +16,10 @@ To install release candidates run `[sudo] gem install cocoapods --pre`
   [Eric Amorde](https://github.com/amorde)
   [#7093](https://github.com/CocoaPods/CocoaPods/pull/7093)
 
+* Show warning when SDK provider tries to push a version with an unencrypted HTTP source
```

Comment:
nit, add two trailing spaces in the end of this line

Reply:
this line could also be updated to the commit message, as the word "prohibit" is not in effect here.